tools/fuchsia/sdk/sdk_targets.gni

```diff
@@ -0,0 +1,139 @@
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/fuchsia/sdk.gni")
+import("//flutter/tools/fuchsia/dart/dart_library.gni")
+import("//flutter/tools/fuchsia/fidl/fidl.gni")
+
+_fuchsia_sdk_path = "//fuchsia/sdk/$host_os"
+
+template("_fuchsia_dart_library") {
+  assert(defined(invoker.meta), "The meta file content must be specified.")
+  meta = invoker.meta
+  assert(meta.type == "dart_library")
+
+  _output_name = meta.name
+  _sources = []
+  _deps = []
+
+  if (defined(meta.dart_library_null_safe) && meta.dart_library_null_safe) {
+    _dart_language_version = "2.12"
+  } else {
+    _dart_language_version = "2.0"
```

Comment:
do we need to support pre null-safe versions of dart?

Reply:
It seems like we do? One example, `fuchsia_vfs` depends on `quiver` v2.1.5, which (according to fuchsia.git's `BUILD.gn`, that is replicated in `flutter/buildroot` `secondary/third_party/pkg/quiver/`) depends on Dart language version 2.0 (not null safe). The Fuchsia SDK includes the `dart_library_null_safe` property, so I don't want to ignore it, even if current Fuchsia Dart SDK libraries always set this to `true` for now.